generators/server/templates/src/main/docker/config/git2consul.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "1.0",
+  "repos" : [{
+    "name" : "config",
+    "url" : "https://github.com/PierreBesson/test-registry.git",
```

Comment:
This is more like a sample config, in before merging, I will change this to `url: jhipster/generator-jhipster.git` and `source_root": "/generators/.../consul-config/`.